Makefile

```diff
@@ -156,6 +156,11 @@ test: $(ENVTEST) manifests generate vet golangci-lint api-docs kustomize-lint he
 	$(info $(M) running $@)
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(BIN) -p path)" go test ./... -coverprofile cover.out
 
+.PHONY: test_short
+test_short: ## go test ./... -test.short
+	$(info $(M) running $@)
+	go test ./... -test.short -coverprofile cover.out
```

Comment:
I normally see `-short`, not `-test.short`. Is it an equivalent?

Reply:
Personally, I prefer `-short` simply because it's used pretty much everywhere => people like me would not be surprised to see the longer version :)